### PR TITLE
Add guidelines for the sprint presentation

### PR DESCRIPTION
**Pull request by ll7**
This pull request includes the following changes:

- Updated sprint review guideline
- Updated markdown settings in .vscode/settings.json

Fixes [Feature]: Add guidelines and an example for the sprint presentation #282

- local linting succeded
- decided to not include an example from paf23, because the guidlines are now much more specific.

## Summary by CodeRabbit

- **New Features**
	- Enhanced Markdown editing capabilities with new settings for table of contents (TOC) levels, automatic TOC updates on save, and unordered list markers.
	- Introduced a new guide for creating effective Sprint Review presentations, detailing structure and style for effective communication.

- **Documentation**
	- Added a placeholder section for "Sprint Planning Guidelines" to the development documentation.

### CODE REVIEW

File: doc/development/sprint_review_presentation.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+- **Interactive**: Ask for feedback, highlight areas that require input or decisions.
+- **Keep it high-level**: Avoid too much technical jargon.
+- **Stay concise**: Respect everyone’s time and keep the presentation focused on what matters most.
+- **Collaborate**: Collaborate with the team to ensure all aspects of the sprint are covered.
+- **Consistent Slide Design**: Use a consistent slide design (fonts, colors, layout) to improve readability.
+  - Templates for AuxMe-Presentations: <https://mediastore.rz.uni-augsburg.de/get/EJ74okFZE2/>
+
+---
+
+next: [Sprint Planning Guidelines](spring_planning_guidelines.md)
```
[the remark this answers is not in the document]
Comment (ll7) [reply doc/development/sprint_review_presentation.md]: Thanks, fixed in https://github.com/una-auxme/paf/pull/339/commits/e9fd09e46e1ca93fb61a1e1a05f71b8275f540f7

File: doc/development/sprint_review_presentation.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Sprint Review Presentation
+
+Creating a good presentation for a **Sprint Review** in PAF is important for communicating the team's achievements, identifying challenges, and planning for future work.
```
Comment (JulianTrommer) [inline doc/development/sprint_review_presentation.md:3]: Should we enforce the template for the wiki files? If so a "**Summary:**" should be added at the beginning of this line.

File: doc/development/sprint_review_presentation.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+To present the sprint progress effectively, especially when dealing with complex software projects, it can be beneficial to split the presentations into two distinct parts:
+
+1. **General Sprint Progress Overview**: This focuses on the big-picture updates, team-wide accomplishments, challenges, and next steps.
+2. **Individual Pull Request (PR) Presentations**: These dive into the details of specific features or changes made during the sprint.
+
+>[!INFO] Table of Contents
```
Comment (JulianTrommer) [inline doc/development/sprint_review_presentation.md:10]: The same would apply to removing this line that is missing in the template.

**Review by JulianTrommer — APPROVED** [review]
Thanks! Looks great to me.

**Pull request merged**